x/bank/keeper/keeper.go

```diff
@@ -355,6 +369,69 @@ func (k BaseKeeper) SendCoinsFromAccountToModule(
 	return k.SendCoins(ctx, senderAddr, recipientAcc.GetAddress(), amt)
 }
 
+
+// DeferredSendCoinsFromAccountToModule transfers coins from an AccAddress to a ModuleAccount.
+// It deducts the balance from an accAddress and stores the balance in a mapping for ModuleAccounts.
+// In the EndBlocker, it will then perform one deposit for each module account.
+// It will panic if the module account does not exist.
```

Comment:
does the entire block get rolled back? IIRC it does, but wanted to double check

Reply:
It would get stuck, but the block will not be committed - in this case we would be able upload a new binary to fix it.